src/main/java/gov/va/ascent/framework/rest/provider/RestProviderHttpResponseCodeAspect.java

```diff
@@ -282,17 +284,20 @@ private void getHttpRequestAuditData(HttpServletRequest httpServletRequest,
 		final String contentType = httpServletRequest.getContentType();
 		
 		if (contentType != null
-				&& (contentType.toLowerCase().startsWith(MediaType.MULTIPART_FORM_DATA_VALUE)
-						|| contentType.toLowerCase().startsWith("multipart/mixed"))) {
+				&& (contentType.toLowerCase(Locale.ENGLISH).startsWith(MediaType.MULTIPART_FORM_DATA_VALUE)
+						|| contentType.toLowerCase(Locale.ENGLISH).startsWith("multipart/mixed"))) {
 			List<String> attachmentTextList = new ArrayList<>();
+			InputStream inputstream = null;
 			try {
 				for (Part part : httpServletRequest.getParts()) {
-					InputStream inputstream = part.getInputStream();
+					inputstream = part.getInputStream();
```

Comment:
@rajuthota1  -- For the same request if you call getInputStream() twice, won't it throw an error. Now these attachments will actually be uploaded by some back end service and for that the code would want to re-read it again. Can you verify it by adding reading the stream again in doc service to make sure that there are no errors doing so..